playbooks/roles/plugins/templates/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+sync_templates: False
```

Comment:
This should probably be renamed to `templates_sync` to follow convention for variable naming.